### ISSUE TITLE
SumatraPDF install seems o disturb Acrobat settings

### ISSUE THREAD

**Issue by GitHubRulesOK** [sumatrapdfreader/sumatrapdf#3891]
I have repeatedly notice that after SumatraPDF installer is run WITHOUT preview support that Adobe Previewer no longer works and Acrobat need the feature to be manually reinstalled, thus appears that having setting off in installer affects wider application support.

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/d8a094dd-6f23-4706-b6b8-4e703c86fd95)

it requires a very very slow double reinstallation
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/9f9357bc-8312-4c95-b0d6-af5ab3c1ff06)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/8baafba9-d1a6-4c52-a3dc-14a26a4af296)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/ed8d14f1-9556-422d-bc7b-fbb1bc2ddfb9)

and even restarting explorer returns a different previewer
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/7f08743c-8092-4d8c-b0c1-95e563bbf092)

**Comment by kjk** [comment]
Closing: not enough information to act on. The report is speculative (installer without preview support may affect Adobe's previewer) with no minimal repro of registry/shell state before vs. after, and no confirmation this still happens on current builds.

If you can still reproduce on a current pre-release, reopen with exact installer options, OS version, and what Adobe setting/handler broke (and ideally before/after registry or Default Apps state).

**Comment by GitHubRulesOK** [comment]
Agreed recent SumatraPDF does not seem to affect Recent Adobe Preview  in fact it is the opposite no matter what I try including tell Acrobat not to preview  (short of uninstall) Adobe will not let go of the preview function, so I guess they hardened it ! taskill host and register sumatrapdf preview and restart host in any order always shows Adobe preview probably until it is not installed on next boot!

<img width="1436" height="536" alt="Image" src="https://github.com/user-attachments/assets/960a0fcb-58a7-4576-a558-d3c23c93fca5" />
